architecture/core_objects/openshift_model.adoc

```diff
@@ -355,7 +355,12 @@ Sample Route:
 ----
 
 == Project
-An OpenShift project allows a community of users to organize and manage their content in isolation from other communities. Each project has its own resources (pods, services, replication controllers, etc.), policies (who can or cannot perform actions), and constraints (this project is allowed this much quota, etc.).  An OpenShift cluster administrator may create an OpenShift project and delegate administrative rights to the project to a member of the user community.
+An OpenShift project allows a community of users to organize and manage their content in isolation from other communities. Each project has its own:
```

Comment:
Could make the first "project" in this sentence be a link to the `dev_guide/projects` topic.